eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package hu.flowacademy.companycalendar.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+@ResponseBody
+public class GlobalExceptionHandler {
+
+  @ExceptionHandler(NotFoundException.class)
+  public ResponseEntity<String> handleNotFoundException(NotFoundException u) {
+    return new ResponseEntity<>(u.getMessage(), HttpStatus.NOT_FOUND);
+  }
+
+  @ExceptionHandler(UserAlreadyExistException.class)
+  public HttpStatus handleBadRequestException(UserAlreadyExistException e) {
+    return HttpStatus.BAD_REQUEST;
```

Comment:
Szerintem itt is adjunk vissza inkább valamilyen response entity-t, amibe bewrappeljük az exception message-ét.